tests/eko/test_ad_nlo.py

```diff
@@ -73,7 +73,37 @@ def test_gamma_1():
         ),
         rtol=6e-7,
     )  # gg
-    gS1 = ad_nlo.gamma_singlet_1(4, NF)
+    gS1 = ad_as2.gamma_singlet(4, NF)
     np.testing.assert_allclose(
         gS1[0, 1], (-56317.0 / 18000.0 * CF + 16387.0 / 9000.0 * CA) * NF
     )  # qg
+
+    update_colors(4)
+    from eko.constants import CA as ca
+    from eko.constants import CF as cf
```

Comment:
Here to get the actual thing updated, the best thing is to keep the scope when imported top level:

```py
from eko import constants as const

assert const.CA == 3
update_colors(4)
assert const.CA == 4
```

If possible, it would be better to do it everywhere, but for the time being if you update here is just fine.